api/routes/InventoryItem.js

```diff
@@ -14,8 +14,10 @@ const {
   FORBIDDEN,
   NOT_FOUND
```

Comment:
this file shouldnt be changed at all, can you refert it back to `dev`? 

do:
`git checkout origin/dev [filename]`

Reply:
This isnt resolved, please double check before